en/docs/reference/mediators/throttle-mediator.md

```diff
@@ -1,22 +1,19 @@
 # Throttle Mediator
 
-The **Throttle Mediator** can be used to restrict access to services.
+The Throttle mediator can be used to restrict access to services.
 This is useful for enterprise-level services to avoid heavy loads that can cause performance issues in the system. 
-It can also be used when you want to avoid certain user groups (i.e. IP addresses and domains) accessing your system. 
+It can also be used when you want to restrict certain user groups (i.e., IP addresses and domains) from accessing your system. 
```

Comment:
```suggestion
It can also be used when you want to restrict certain user groups (for example, IP addresses and domains) from accessing your system. 
```